### DISCUSSION TITLE
Feature request: New option to display the 'user friendly name' in the WebUI

### DISCUSSION THREAD

**Discussion by faberlau** [arendst/Tasmota#17921] [Ideas]
Hello Team,

As far as I know, there are currently two options to control the display of information in the WebUI: SO53 (hostname) and SO141 (model name).

I would like to have another option to display the 'user friendly name' also in the WebUI.

BR Faberlau

**Comment by sfromis** [comment]
You could set the DeviceName to the same string, when you only have one FriendlyName
